COMMERCE-12452 - Console errors appear after upgrading from 7.2 then manually reindexing assets

victorcdp opened the pull request:
Related Issue: [COMMERCE-12452](https://liferay.atlassian.net/browse/COMMERCE-12452)

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

noornajj commented:
ci:test:sf

noornajj commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-11.liferay.com/job/test-portal-source-format/4863/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-11.liferay.com//userContent/jobs/test-portal-source-format/builds/4863/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 1 minute</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b50d81b60bf1437cc1418227d8d96eaefefce314">b50d81b60bf1437cc1418227d8d96eaefefce314</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/noornajj/liferay-portal/tree/LPS-123392">LPS-123392</a><br/>Branch GIT ID: <a href="https://github.com/noornajj/liferay-portal/commit/05c3a8952e666e1607fa7e92f765b1fb07c29f23">05c3a8952e666e1607fa7e92f765b1fb07c29f23</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-11.liferay.com/job/test-portal-source-format/4863/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-11.liferay.com/userContent/jobs/test-portal-source-format/builds/4863/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-11.liferay.com/job/test-portal-source-format/4863/">test-portal-source-format#4863</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-11.liferay.com/userContent/jobs/test-portal-source-format/builds/4863/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> sf<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/120">brianikim#120</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/1302.aspx">/Liferay DXP 7.3/Pull Request/ci:test:sf</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/60016.aspx">master - noornajj > brianikim - PR#120 - 2020-11-11[14:24:21]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-1.liferay.com/job/publish-spira-report/13434/">publish-spira-report#13434</a><br />

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/7924/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-11.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7924/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 9 out of 9 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 22 out of 23 jobs passed in 3 hours 31 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b50d81b60bf1437cc1418227d8d96eaefefce314">b50d81b60bf1437cc1418227d8d96eaefefce314</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/8d22a791300569a6c92e7b9152e6ae1c6bdc863a">8d22a791300569a6c92e7b9152e6ae1c6bdc863a</a></p><details><summary><strong>ci:test:stable - 9 out of 9 jobs PASSED</strong></summary><details><summary><strong>9 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/468900/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/455668/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/450514/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/370666/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/468901/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/370667/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/478449/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/446509/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/396495/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 21 out of 23 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/7924/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/455667/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>21 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/320602/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/468902/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/468900/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/446511/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/478451/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/455668/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/450514/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/427984/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/478450/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/446510/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/396496/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/370666/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/468903/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/468901/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/370665/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/370667/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/478449/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/468904/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/446509/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/370664/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/396495/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-11.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7924/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at b50d81b60bf1437cc1418227d8d96eaefefce314:</strong></summary><ol><li><div><h5><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/455667/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/455667//consoleText">AXIS_VARIABLE=0,label_exp=!master #455667</a><div><p>Please fix <strong>semantic versioning</strong> on <strong><a href="https://github.com/noornajj/liferay-portal/tree/LPS-123392">noornajj/LPS-123392</a></strong><pre><code>     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.frontend.taglib.clay.data.set.servlet.taglib.util MINOR      1.0.0      1.0.0      1.1.0      VERSION INCREASE REQUIRED
     [exec] 	&lt;   class      com.liferay.frontend.taglib.clay.data.set.servlet.taglib.util.ClayDataSetActionDropdownItem
     [exec] 		+   method     setType(java.lang.String)
     [exec] 			+   return     void
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] 
     [exec] &gt; Task :apps:frontend-taglib:frontend-taglib-clay:syncVersions
     [exec] 
     [exec] &gt; Task :apps:frontend-taglib:frontend-taglib:compileJSP
     [exec] Note: Some input files use or override a deprecated API.
     [exec] Note: Recompile with -Xlint:deprecation for details.
     [exec] Note: Some input files use unchecked or unsafe operations.
     [exec] Note: Recompile with -Xlint:unchecked for details.
     [exec] 
     [exec] &gt; Task :apps:frontend-taglib:frontend-taglib:jar
     [exec] &gt; Task :apps:frontend-taglib:frontend-taglib:autoUpdateXml SKIPPED
     [exec] &gt; Task :apps:frontend-taglib:frontend-taglib:baseline
     [exec] &gt; Task :apps:frontend-taglib:frontend-taglib:syncVersions
     [exec] Gradle build finished at 2020-11-12 03:55:13.016.
     [exec] 112 actionable tasks: 104 executed, 8 up-to-date
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:frontend-taglib:frontend-taglib-clay:baseline'.
     [exec] &gt; A failure occurred while executing com.liferay.gradle.plugins.baseline.internal.work.BaselineWorkAction
     [exec]    &gt; org.gradle.api.GradleException: Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/tools/sdk/dist/com.liferay.frontend.taglib.clay-7.1.0.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay/com.liferay.frontend.taglib.clay/6.3.3/ed31b19ea17cec50d62438233bc3693fa30d6629/com.liferay.frontend.taglib.clay-6.3.3.jar</code></pre></p></div></div></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/7924/">test-portal-acceptance-pullrequest(master)#7924</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-11.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7924/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> relevant<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/120">brianikim#120</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/1312.aspx">/Liferay DXP 7.3/Pull Request/ci:test:relevant</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/60144.aspx">master - noornajj > brianikim - PR#120 - 2020-11-11[16:49:07]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-16.liferay.com/job/publish-spira-report/11047/">publish-spira-report#11047</a><br />

izaera committed:
LPS-192879 Emit the CSP nonce from .tpl templates

izaera committed:
LPS-192789 Don't propagate nonce to browser if strict-dynamic is active

When strict-dynamic is active there's no need to send the nonce to the client
side because the trust is propagated automatically. This may be used by customers
that deem propagating the nonce to the browser insecure.

izaera committed:
LPS-192879 Change the way we handle the nonce

1. We always store a valid nonce in the session
2. We only set the TLS nonce when CSP is active
3. When getting the nonce without request we return the one in the TLS
4. When requesting the nonce with a request we return the one from session or ""
   if CSP is disabled.

izaera committed:
LPS-192879 Don't store nonced responses in the cache

Since they only make sense for a single user.

achaparro committed:
LPS-196393 Use random companyIds to avoid collisions in LXC

achaparro committed:
LPS-196393 Tests: add a check anytime we create a company to be sure it is random and we do not use the counter

Commit:
LPS-198502 Replace ServiceProxyFactory with Snapshot in class ServiceAccessPolicyManagerUtil

alessiorendina committed:
COMMERCE-12306 Added new localized keys

alessiorendina committed:
COMMERCE-12306 BuildLang

alessiorendina committed:
COMMERCE-12306 Added the feature flag

alessiorendina committed:
COMMERCE-12306 Added new constants

alessiorendina committed:
COMMERCE-12307 Added required dependency

alessiorendina committed:
COMMERCE-12306 Regen services

alessiorendina committed:
COMMERCE-12306 Added new method to commerce order rule entry local and remote service implementation

alessiorendina committed:
COMMERCE-12306 Fixed commerce order rule entry type and registry

alessiorendina committed:
COMMERCE-12306 Added new commerce order rule entry type and jsp contributor implementation

alessiorendina committed:
COMMERCE-12306 Added new dataset

alessiorendina committed:
COMMERCE-12306 Added bi-directionality support

alessiorendina committed:
COMMERCE-12306 Refactor display contexts

alessiorendina committed:
COMMERCE-12306 Fixed action command

alessiorendina committed:
COMMERCE-12306 Semantic versioning (gw baseline)

alessiorendina committed:
COMMERCE-12307 Added new fields to product option value schema

alessiorendina committed:
COMMERCE-12306 Added new commerce product definition link types implementation

alessiorendina committed:
COMMERCE-12307 Regen REST module

alessiorendina committed:
COMMERCE-12307 Refactor product option value dto converter

alessiorendina committed:
COMMERCE-12306 Added new dto

alessiorendina committed:
COMMERCE-12306 Auto SF (copyright check)

davisaints committed:
LPS-198440 Add object validations via API to make tests faster

robertoDiaz committed:
LPS-199272 Handle the I18nPath if necessary

alessiorendina committed:
COMMERCE-12306 Extended commerce order rule entry type

# breaking_change_report

## What modules/apps/commerce/commerce-order-rule-api/src/main/java/com/liferay/commerce/order/rule/entry/type/COREntryType.java

Method evaluate(COREntry corEntry, List<COREntryTypeItem> corEntryTypeItems) is added

## Why

This change is made to evaluate if a rule can be applied before creating an order

alessiorendina committed:
COMMERCE-12306 Removed no more necessary files

# breaking_change_report

## What modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/configuration/CPDefinitionLinkTypeSettings.java modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/configuration/CPDefinitionLinkTypeConfigurationWrapper.java modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/configuration/CPDefinitionLinkTypeSettingsImpl.java

Classes are deleted

## Why

This change is made because this is a duplicate of modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/links/CPDefinitionLinkTypeRegistry.java

smottal committed:
COMMERCE-12227 Add module dependencies

smottal committed:
COMMERCE-12227 New keyword

smottal committed:
COMMERCE-12227 Update DisplayContext

smottal committed:
COMMERCE-12227 New modal

smottal committed:
COMMERCE-12227 Autogenerated Code (buildLang)

davisaints committed:
LPS-198440 Put the property on the right line

Commit:
LPS-198502 Replace ServiceProxyFactory with Spring in class TunnelAuthenticationManagerUtil, as TunnelAuthenticationManagerImpl is a spring bean

smottal committed:
COMMERCE-12227 Code review requested changes

victorhcunha committed:
LPS-199545 Add path to select object in sidebar

Commit:
LPS-198961 Get 'status' properties value

Commit:
LPS-198961 Check if status is approved to delete the custom object

brianchandotcom committed:
LPS-196393 Sort

Commit:
LPS-198502 Semantic versioning

lfbesada committed:
LPS-200227 Throws a LockedLayoutException if a user tries to change the classNameId or the classTypeId of a locked page

smottal committed:
COMMERCE-12227 Remove loop

pei-jung committed:
LPS-200278 Update version as long as it's in draft status

pei-jung committed:
LPS-200278 If the current structure version is a draft, update it instead of adding a new one

lfbesada committed:
LPS-200227 SF use the LayoutLockManager method

lfbesada committed:
LPS-200227 Adds new getLock signature to specify the layout and the user ID instead of get them from theme display

lfbesada committed:
LPS-200227 Prevents NPE if layout is null

lfbesada committed:
LPS-200227 Throws an UnsupportedOperationException if trying to change the content type and the feature flag is disabled

ling-alan-huang committed:
LPS-200296 Enforce name of class extending InitialRequestPortalInstanceLifecycleListener

achaparro committed:
LPS-196393 Not needed since an exception will be thrown if the generated companyId already exists

izaera committed:
LPS-192879 Emit the CSP nonce from Java code

izaera committed:
LPS-192879 Emit the CSP nonce from .ftl and .vm templates

izaera committed:
LPS-192879 Emit the CSP nonce from .tmpl templates

rafaprax committed:
LPS-199912 Create upgrade to change storage format of the workflow definitions content from xml to json

rafaprax committed:
LPS-199912 Move format checking code to the WorkflowDefinitionContentUtil

rafaprax committed:
LPS-199912 Make sure definition content will be stored as json

rafaprax committed:
LPS-199912 Add a parameter in the workflow definition headless api to know if the content should be returned in xml or json format. It was required because the workflow UI use the headless API and it must still receiving the content in XML format

rafaprax committed:
LPS-199912 Build Rest

rafaprax committed:
LPS-199912 Update tests

lfbesada committed:
LPS-200227 Take the draft layout lock before changing the content-type and release it after performing

JanisZhang committed:
LPS-200625 Remove unused Reference field _mailService as its usage was removed by f94a15de03d1047a0a7df7cd7f1b70ee40158bef

JanisZhang committed:
LPS-200625 Remove unused Reference field _userLocalService as its usage was removed by 12802fb5f9f2429e7c7927700855475f669c2b7d

JanisZhang committed:
LPS-200625 Remove unused Reference field _objectFieldLocalService as its usage was removed by 3e1aa1461e0248f23008100b8e5daa9c0b43f2ce

JanisZhang committed:
LPS-200625 Remove unused Reference field _panelAppRegistry as its usage was removed by d70092efe256ccbf84b1d32975589a6c2f306eef

sandrodw3 committed:
LPS-200227 Consider isLocked error

DiegoHu97 committed:
LPS-200741 Replace AUI() in dynamic-data-mapping-form-web

baharturk committed:
COMMERCE-12227 Refactor tests according to the new functionality

sandrodw3 committed:
LPS-200227 buildLang

Commit:
LPS-200081 Fix the test, as the field _authToken is no longer included in AuthTokenUtil, and we want to completely replace ServiceProxyFactory with Snapshot, so modify the test to testSnapshotIntegrationPoint

izaera committed:
LPS-192789 Extract common code to Util class

izaera committed:
LPS-192789 Cache CSP configuration in request

Since obtaining the CSP configuration is a somewhat costly operation that we are
doing multiple times per request, we store it in the request so that we retrieve
it just once.

ling-alan-huang committed:
LPS-200296 Rename

rafaprax committed:
LPS-199912 Create a method into the KaleoDefinitnion*Impl and WorkflowDefinition classes to avoid call the converter logic more than once to the same entity into the same call stack

rafaprax committed:
LPS-199912 Build service

rafaprax committed:
LPS-199912 Baseline

rafaprax committed:
LPS-199912 Apply to current usages that need the content to be in xml format

rafaprax committed:
LPS-199912 Use the high level methods instead of parse the workflow definition content

markomihalovics committed:
LPS-200885 Logical order changes in the FriendlyURLServlet.getRedirect method to prevent unauthenticated users from accessing private URL pages and to handle LayoutPermissionExceptions properly. Additionally, removed a redundant if statement.

markomihalovics committed:
LPS-200885 Code format in the FriendlyURLServlet.

Commit:
LRQA-83375 Add assignRoleToUser JSON macro to setup

lfbesada committed:
LPS-200191 New thread local for updating the layout status

lfbesada committed:
LPS-200191 Baseline

lfbesada committed:
LPS-200191 New thread local for updating the layout status

lfbesada committed:
LPS-200191 Prevent from turn the layout status to draft when updating its LayoutPageTemplateStructure or its FragmentEntryLinks if isUpdateLayoutStatus is disabled

JLucasM committed:
LRAC-14877 Support downloading PDF report for Blogs Overview

joelrocha68 committed:
ISSD-212 Add path to CP2UserProfileMenu

joelrocha68 committed:
ISSD-212 Add macro

joelrocha68 committed:
ISSD-212 Add AssertThatUserCanOutLoggedOfSite test

joelrocha68 committed:
ISSD-212 Add AssertThatClickOnAccountSettingsIsRedirectedToLoginPage test

joelrocha68 committed:
ISSD-212 Delete CP2ProjectSelection testcase

Commit:
LRQA-83375 Edit tasks and add JSON macros to testcase

Commit:
LRQA-83375 SF

larissacribeiro committed:
LPS-195395 Move aria-required to the right elements

achaparro committed:
LPS-196393 Agreed with LXC to set a higher bottom limit to reserve specific range for companyIds and facilitate the virtual instance replication inside the same environment

izaera committed:
LPS-192879 baseline

baharturk committed:
COMMERCE-12227 Add a new macro

baharturk committed:
COMMERCE-12227 Add a new path

lfbesada committed:
LPS-200227 Handles LockedLayoutException if occurs performing the action

lfbesada committed:
LPS-200227 Adds a service method to get a LayoutPageTemplateEntry

lfbesada committed:
LPS-200227 BuildService

lfbesada committed:
LPS-200227 Use the new method to GET a LayoutPageTemplateEntry in all those places that are using the FETCH but assuming that LayoutPageTemplateEntry won't be null

jkappler committed:
LPS-201001 Hide behind FF

jkappler committed:
LPS-201001 Adds FF to portal.properties

IstvanD committed:
LPS-201025 The locale may not be available

joelrocha68 committed:
ISSD-212 Fix

riccardo-ferrari committed:
LRAC-14782 analytics-dxp-entity-rest-impl Add UserContentRecommendation model

riccardo-ferrari committed:
LRAC-14782 analytics-dxp-entity-rest buildREST

Commit:
ISSD-1509 Create License Details Modal

Commit:
ISSD-1555 Changing Contact Us footer

MumenTayyem committed:
LPS-197318 - Ticket Actions and Relationships

joelrocha68 committed:
ISSD-212 Test Fix

MumenTayyem committed:
LPS-197318 - Ticket Actions and Relationships

lfbesada committed:
LPS-200191 Adds a model listener to unmap the context references from LayoutPageTemplateStructure data when the mapping types have changed

smottal committed:
COMMERCE-12227 SF

lfbesada committed:
LPS-200191 Updates the FragmentEntryLinks editableValues

lfbesada committed:
LPS-200191 Uses UpdateLayoutStatusThreadLocal to prevent updating the layout status during the process

joelrocha68 committed:
LPS-200040 Add clickInTextLink macro

joelrocha68 committed:
LPS-200040 Add AssertThatAlertAppearsWhenAddTwoTeamMemberWithNewDomain test

joelrocha68 committed:
LPS-200040 Add AssertThatTheAdmCanAddTeamMemberWithNewDomain test

joelrocha68 committed:
LPS-200040 Add AssertThatTheLinkRedirectedToOktaLoginPage test

rafaprax committed:
LPS-199912 No need to create 2 companies to test the upgrade. Use the default company instead to speed up test execution

boton committed:
LPS-200742 Add validation to current date format

boton committed:
LPS-200742 Add some frontend tests around isValidDate function

boton committed:
LPS-200742 Use a const for validation and config the DatePicker avoiding magic numbers

JLucasM committed:
LRAC-14879 Support downloading PDF report for Documents and Media Overview

smailbiala committed:
LPS-197192 add a label element to the preselected values

lfbesada committed:
LPS-200227 Baseline

boton committed:
LPS-200660 Fix the timezone ClayDatePicker expect timezone instead of timeZone, add prop validation

rafaprax committed:
LPS-199912 Rename parameter

manuelRives committed:
LPS-199728 - Remove duplicated close button.

sergiojimcos committed:
LPS-200646 Fix table-layout property to eshow correctly the content of a base64 file content

JLucasM committed:
LRAC-14884 Support downloading PDF report for Form Overview

smailbiala committed:
LPS-197192 SF

boton committed:
LPS-200742 Trim the callback date too

smailbiala committed:
LPS-197192 Add aria-label attribute to enhance checkbox accessibility

MumenTayyem committed:
LPS-197318 - Adding Actions/Relationships

MumenTayyem committed:
LPS-197318 - Adding Actions/Relationships (CreateStrategy=INSERT)

dsanz committed:
LPS-201102 Avoid holding a reference to ClientExtensionEntry model impl in the CET. This makes portlet registration process to not keep that reference

riccardo-ferrari committed:
LRAC-14782 analytics-machine-learning-impl Download and Import UserContentRecommendations

riccardo-ferrari committed:
LRAC-14782 analytics-machine-learning-impl Download and Import UserContentRecommendations resources

pei-jung committed:
LPS-200278 Update test to reflect the change: when a form in draft status is updated, update the existing version instead of creating a new one

pei-jung committed:
LPS-200278 Rename test to make the test case clear

fmarcio committed:
LRAC-14894 Implement Download Report button on web content

Commit:
ISSD-1580 Fix Navigation menu and footer padding

Commit:
ISSD-1580 Fix padding in dashboard pages

Commit:
ISSD-1576 Fix Delete button not displayed for CMM and CMD

Co-authored-by: caiosouza15 <caio.souza@liferay.com>

larissacribeiro committed:
LPS-199072 Update snapshots

michaelhashimoto committed:
LRCI-3820 Upgrade activemq 5 to activemq 6 to allow passwords to be set

michaelhashimoto committed:
LRCI-3820 Customize the Dockerfile to include the docker-run.sh

michaelhashimoto committed:
LRCI-3820 SF

michaelhashimoto committed:
LRCI-3820 Add logic to configure activemq with clustering

michaelhashimoto committed:
LRCI-3820 Remove unnecessary load balancer port

michaelhashimoto committed:
LRCI-3820 Rename from liferay-jethr0-sidecar-activemq to liferay-jethr0-sidecar-jms-master to leave room for another image

Commit:
LPS-201148 Replace ServiceProxyFactory with dynamic Snapshot in class PermissionCheckerFactoryUtil, as based on PTR-2102 and LPS-78573, I think we have customers who customize their own implementation of interface PermissionCheckerFactory, so we are supposed to use dynamic snapshot in class PermissionCheckerFactoryUtil.

kenjiheigel committed:
LRCI-3868 Use jsdelivr cdn for static frontend files

joelrocha68 committed:
LPS-200040 Test Fix

JessieZhao0714 committed:
LRQA-84188 Update test cases due to DDL being deprecated

pei-jung committed:
LPS-200278 Update structureVersion's definition

Commit:
LRQA-84232 Test fix to add unique in objectField in BatchEngineBrokerTest.testImportObjectDefinition

ealonso committed:
LPS-201173 Removes LPS-197408 Feature Flag

ealonso committed:
LPS-201174 Removes LPS-197307 Feature Flag

MumenTayyem committed:
LPS-197318 - Adding Users

lfbesada committed:
LPS-200191 Prevents from process the inputs fragmentEntryLinks

oliverkecskemety committed:
LPS-201158 Use feature flag properly on company scope

brianchandotcom committed:
LPS-200966 Regen

liulukiwi committed:
LPS-200878 Add version for javax.servlet

dsanz committed:
LPS-201102 Fix test main component name for the client extension suite

brianchandotcom committed:
LPS-200278 Can we do this?

Mikellorza committed:
LPS-201172 Take into account the scope site connected group ids to show categories and tags

brianchandotcom committed:
LPS-192879 Simplify

brianchandotcom committed:
LPS-192879 Good enough

brianchandotcom committed:
LPS-197318 Rename

ivicac committed:
LRAC-14630 - Rename “Origin-” to “Liferay-Origin-”

MumenTayyem committed:
LPS-197318 - Adding Actions & Relationships

brianchandotcom committed:
LPS-197318 Auto SF

brianchandotcom committed:
LPS-197318 SF

izaera committed:
LPS-192879 Rename nonceAttr to nonceAttribute

brianchandotcom committed:
LPS-197318 Use prefix

Commit:
ISSD-1580 Fix margin top for dashboard pages

Commit:
ISSD-1580 Change dashboard page type

Commit:
ISSD-1580 SF

izaera committed:
LPS-192879 Simplify

brianchandotcom committed:
LPS-199912 Rename

brianchandotcom committed:
LPS-199912 auto SF

brianchandotcom committed:
LPS-199912 Rename

Commit:
ISSD-1556 Add requests to license keys API and Table Pagination

brianchandotcom committed:
LPS-199912 Rename

brianchandotcom committed:
LPS-199912 auto SF

brianchandotcom committed:
LPS-199912 Regen

brianchandotcom committed:
LPS-199912 As used

brianchandotcom committed:
LPS-199912 SF

brianchandotcom committed:
LPS-199912 Is this right?

brianchandotcom committed:
COMMERCE-12307 SF

Commit:
ISSD-1580 Fix dashboard margin bottom

achaparro committed:
LPS-201102 We should not flush the session if we do not change the companyId

alessiorendina committed:
COMMERCE-12306 Rename

victorhcunha committed:
LPS-199545 Add CanSearchForObjectDefinition test

brianchandotcom committed:
analytics-batch-export-import-impl 3.0.7 prep next

brianchandotcom committed:
data-engine-js-components-web 1.0.117 prep next

brianchandotcom committed:
product-navigation-user-personal-bar-web 6.0.20 prep next

pedroliveirav committed:
ISSD-1541 Fix unnecessary API calls errors on the project page

brianchandotcom committed:
dynamic-data-mapping-service 7.1.18 prep next

brianchandotcom committed:
object-service 1.0.174 prep next

brianchandotcom committed:
object-web 1.0.187 prep next

brianchandotcom committed:
headless-builder-impl 1.0.19 prep next

Commit:
ISSD-1509 Adjust Custom Field Value

Commit:
ISSD-1509 Fix Account Information Component

Commit:
ISSD-1509 Improving Footer Buttons Component

Commit:
ISSD-1509 Fix Create License Form Layout

Commit:
ISSD-1556 Better formated values to match Figma

Commit:
ISSD-1509 Improving Product Card Component

Commit:
ISSD-1509 Create License Key Details Modal

brianchandotcom committed:
LRCI-3868 prep next

Commit:
ISSD-1577 Keys deactivation modal structure

brianchandotcom committed:
LPS-192879 SF

kevenleone committed:
ISSD-1580 Fix Navigation

kevenleone committed:
ISSD-1580 Move directory

Commit:
ISSD-1580 Fix padding in dashboard pages

kevenleone committed:
ISSD-1580 Update Routes and Params

michaelhashimoto committed:
LRCI-3870 Avoid concurrent modification error for reinvoke rules

ealonso committed:
LPS-201117 Sets necessary vars to ThemeDisplay

dsanz committed:
LPS-201102 Follow the same pattern (first persist, then deploy the client extension) in updateStatus method, just like the other service methods. This also fixes the DB persistence issue because there are no deployment operations in the middle of an update

ealonso committed:
LPS-201117 Sets PermissionCheckerMethodTestRule

ealonso committed:
LPS-201117 Checks correct type

ealonso committed:
LPS-201179 We need to check if the folder id of the article is the parent folder or not since if it is the parent folder the getFolder method is returning an empty folder causing undesired values for breadcrumb

Commit:
ISSD-1580 Fix margin for version and status in app dashboard

kevenleone committed:
ISSD-1556 SF

ealonso committed:
LPS-201037 Uses correct redirect

shuyangzhou committed:
LPS-201267 Avoid assignable checking for well known classes.

brianchandotcom committed:
LPS-201102 Rename

riccardo-ferrari committed:
LRAC-14782 analytics-machine-learning-impl Rename batch engine task item delegate

michaelhashimoto committed:
LRCI-3820 Add the credentials to connections made to JMS

michaelhashimoto committed:
LRCI-3820 Rename properties from activemq to jms

michaelhashimoto committed:
LRCI-3820 Add jenkins master name as a property filter

pyoo47 committed:
LRCI-3872 Eliminate a constant that is only used in one place.

Commit:
LRQA-84258 Modify macros to support enable lazy loading

Commit:
LRQA-84258 Modify cases to enable lazy loading

austinchiang committed:
LPS-201169 Baseline

Commit:
LPS-201276 Remove deprecated case

Commit:
LPS-201278 Fix typo

Commit:
LPS-201277 Modify cases to adapt new markup

Commit:
LPS-201280 Modify case to adapt markup

Commit:
LPS-201281 Modify locator to adapt new markup

brianchandotcom committed:
LRCI-3870 LRCI-3872 prep next

brianchandotcom committed:
LPS-196393 Rename

Commit:
COMMERCE-12452 prevent NPEs

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ee303dbf5d7a3c23dfc86c727bb4c78ed894e749">ee303dbf5d7a3c23dfc86c727bb4c78ed894e749</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-12452">COMMERCE-12452</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/ee9aa97de569f98ecf62d46fa7bf63f456679f8b">ee9aa97de569f98ecf62d46fa7bf63f456679f8b</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-source-format/5574/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-source-format/builds/5574/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-7.liferay.com/job/test-portal-source-format/5574/">test-portal-source-format#5574</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-source-format/builds/5574/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/120">brianikim#120</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2455929504">[master] ci:test:sf - victorcdp &gt; brianikim - PR#120 - 2023-11-09[10:35:07]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-7.liferay.com/job/publish-testray-report/3394/">publish-testray-report#3394</a><br/></div></div>

brianikim commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ee303dbf5d7a3c23dfc86c727bb4c78ed894e749">ee303dbf5d7a3c23dfc86c727bb4c78ed894e749</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-12452">COMMERCE-12452</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/0e426f6b1bd73045cd5736aaed046de62694d96e">0e426f6b1bd73045cd5736aaed046de62694d96e</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-15.liferay.com/job/test-portal-source-format/4986/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-15.liferay.com/userContent/jobs/test-portal-source-format/builds/4986/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-15.liferay.com/job/test-portal-source-format/4986/">test-portal-source-format#4986</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-15.liferay.com/userContent/jobs/test-portal-source-format/builds/4986/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/120">brianikim#120</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2455930190">[master] ci:test:sf - victorcdp &gt; brianikim - PR#120 - 2023-11-09[10:48:53]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-15.liferay.com/job/publish-testray-report/4289/">publish-testray-report#4289</a><br/></div></div>
